.github/workflow/csi_operator_ci.yaml

```diff
@@ -0,0 +1,226 @@
+name: Operator test
+
+env:
+  OP_TEST_DEBUG: 1
+  OP_TEST_CONTAINER_OPT: "-t"
+on:
+  push:
+    pull_request:
+      - develop
```

Comment:
should we add it to master as well?

Reply:
I thought about it, and before we will merge to master, the code will already be checked because there won't be changed in the code from the last PR to develop and from the PR to master.
But it doesn't cost much to add it for extra safety, so I can add it

Reply:
thanks

Reply:
done